src/components/core/Button/DeprecatedButton.tsx

```diff
@@ -0,0 +1,107 @@
+import { memo, useMemo } from 'react';
```

Comment:
This whole file is actually a renamed from the previous `Button.tsx` but the diff isn't making that easy to see. Maybe best to look at each commit.